src/main/java/teammates/ui/webapi/request/FeedbackSessionRemindRequest.java

```diff
@@ -0,0 +1,45 @@
+package teammates.ui.webapi.request;
+
+/**
+ * The request of specifying reminder emails to be sent.
+ */
+public class FeedbackSessionRemindRequest extends BasicRequest {
```

Comment:
Should we put the request object into another package as it will incorrectly be synced to the frontend.

Reply:
I don't see this as a good reason. Yes it will be synced to the front-end, but it is not incorrect unless someone decides to incorrectly use it (which should be caught during E2E test).